gems/pending/appliance_console/logging.rb

```diff
@@ -57,17 +57,15 @@ def self.log_filename
 
     # TODO: move say_error and say_info to prompting module?
     def say_error(method, output)
-      if interactive?
-        log = "\nSee #{ApplianceConsole::Logging.log_filename} for details." if ApplianceConsole::Logging.log_filename
-        text = "#{method.to_s.humanize} failed with error - #{output.truncate(200)}.#{log}"
-        say(text)
-        press_any_key
-        raise MiqSignalError
-      end
+      log = "\nSee #{ApplianceConsole::Logging.log_filename} for details." if ApplianceConsole::Logging.log_filename
+      text = "#{method.to_s.humanize} failed with error - #{output.truncate(200)}.#{log}"
+      say(text)
```

Comment:
My only concern is if it's ok to say in non-interactive mode... I believe the tests were run in non-interactive mode... Also, I believe @dajohnso's team uses appliance_console_cli to automate some of their workflows so if we write new messages to stdout, I want to make sure they know about it.  They might be parsing stdout.